samples/highcharts/chartchooser/sunburst-monochrome/demo.html

```diff
@@ -0,0 +1,6 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/export-data.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
```

Comment:
Missing the sunburst module (fails in utils)

Reply:
Done